sentinel-core/src/main/java/com/alibaba/csp/sentinel/node/metric/MetricTimerListener.java

```diff
@@ -40,9 +41,13 @@
     public void run() {
         Map<Long, List<MetricNode>> maps = new TreeMap<>();
         for (Entry<ResourceWrapper, ClusterNode> e : ClusterBuilderSlot.getClusterNodeMap().entrySet()) {
-            ClusterNode node = e.getValue();
-            Map<Long, MetricNode> metrics = node.metrics();
-            aggregate(maps, metrics, node);
+            if (SentinelConfig.shouldFilterMetric() && RuleProvider.isNotInRules(e.getValue().getName())) {
+                RecordLog.debug("[MetricTimerListener] Metric filter by config ", e.getValue());
```

Comment:
The argument of `debug` method is incorrect, may change to`RecordLog.debug("[MetricTimerListener] Metric filter by resourceName={}", e.getValue().getName());`

Reply:
Fixed in #2209